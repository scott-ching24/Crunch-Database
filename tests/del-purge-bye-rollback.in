set a 1 2
set b 2 3
snapshot
del b
list entries
purge a
list entries
rollback 1
list entries
bye
