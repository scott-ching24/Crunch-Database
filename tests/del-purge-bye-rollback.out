> ok

> ok

> saved as snapshot 1

> ok

> a [1 2]

> ok

> no entries

> ok

> b [2 3]

> bye
